include/TRFFitter.h

```diff
@@ -3,8 +3,8 @@
 
 #include <vector>
 
-#include "TNamed.h"
-#include "Rtypes.h"
+#include <TNamed.h>
+#include <Rtypes.h>
```

Comment:
I'm not arguing against implementing this, I agree this DOES remove FALSE coincidence multiplicity but GENUINE multiplicity>1 still exist.
Just as a front vs back energy gate that is totally standard for cleaning dssd data.
But that is the next stage, I was just trying to get this code up to at least as functional as the master branch.